oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/SecureFacetTest.java

```diff
@@ -172,6 +172,51 @@ private void createLargeDataset() throws RepositoryException {
         assertNotEquals("Acl-ed and actual counts mustn't be same", actualLabelCount, actualAclLabelCount);
     }
 
+    private void createSmallDataset() throws RepositoryException{
```

Comment:
I haven't read the test, but please do a quick check of existing tests that they are working with "larger than threshold" data set...

Reply:
Ran all the tests - they were all green .

Reply:
no, i didn't mean if they pass or not... with this issue fixed, lower result size would still give "right" results expected from a statistical pov... I think we should make sure that existing tests create/get-results which are bigger that sample size threshold.

Reply:
ok... ignore the noise I created... just read the tests again and found that `createLargeDataset` already is creating num_labels * sample_size nodes (and including all of them in results)... I just wasn't sure if that was ensure already or not.